hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/UnsupportedContentTypeException.java

```diff
@@ -22,4 +23,12 @@ public UnsupportedContentTypeException(Subscription subscription) {
         ));
     }
 
+    public UnsupportedContentTypeException(String payloadContentType, Schema schema) {
```

Comment:
That's right. In most cases schema name matches topic name, and tests didn't catch that typo.
Fixed